mediation_fees.rst

```diff
@@ -0,0 +1,89 @@
+Raiden Mediation Fee Specification
+##################################
+
+Overview
+========
+
+Mediation fees are used to incentivize people to mediate payments. There are (currently) three components:
+
+- Flat fees
+- Proportional fees
+- Imbalance penalty fees
+
+TODO: expand
+
+Fee calculation
+===============
+
+In this description a simple mediation node is assumed. The incoming channel (for the mediated payment) is also called **payer channel** and the outgoing channel is also called **payee channel**. This scenario is shown in the following graphic:
+
+::
+
+               c
+    --> a --> (M) --> b -->
+
+- ``a`` is the locked amount of the payer channel. This locked amount includes fees.
+- ``b`` is the locked amount of the payee channel. This locked amount includes fees for further hops.
+- ``c`` is a helper value for making the calculation of mediation fees simpler. It is not exposed to the user.
+
+
+In the calculation the different fees are used.
+
+- :math:`f` is the flat fee
+- :math:`i` is the imbalance fee
+- :math:`q` is the *per-channel* imbalance fee (see below)
+
+
+Converting per-hop proportional fees in per-channel proportional fees
+=====================================================================
+
+User usually think about mediation fees as a atomic actions, so it's easier to
+let them define a *per-hop* proportional mediation fee (called :math:`p`).
+However, this setting then needs to be converted into a *per-channel*
+proportional fee (called :math:`q`).
+
+.. math::
+
+    b(1+p) = b + bq + qb(1+p)  \\
+    (1+p) = 1 + q + q(1+p)  \\
+    q + q(1+p) = p \\
+    q(2+p) = p \\
+    q = \frac{p}{2+p}
+
+Forward calculation (as in the client)
```

Comment:
for every fee combined now, right? so maybe make this with the same heading-level as the last chapter